com.unity.ml-agents.extensions/Runtime/Input/Adaptors/ButtonInputActionAdaptor.cs

```diff
@@ -30,7 +31,14 @@ public ActionSpec GetActionSpecForInputAction(InputAction action)
         public void QueueInputEventForAction(InputAction action, InputControl control, ActionSpec actionSpec, in ActionBuffers actionBuffers)
         {
             var val = actionBuffers.DiscreteActions[0];
-            InputSystem.QueueDeltaStateEvent(control, (byte)val);
```

Comment:
This is where I was getting a (cannot convert byte to bit) exception, and I changed it to the code below